con-base/src/main/java/ba/codecentric/base/domain/Hotel.java

```diff
@@ -62,6 +57,14 @@
     @Size(max = 500, message = "Description is too long, 500 characters allowed")
     private String description;
 
+    @NotNull(message = "Please add Check-in time")
+    @Column(name = "check_in")
+    private Time checkIn;
```

Comment:
`LocalTime` will be saved as a small blob and remain in its full format. Note that it can be always reformatted. Maybe the idea of modifying getters and setters is not the best. Additional methods for formatting/parsing might be the way to go. I just pointed that you'd need some conversion between your input time (string to `LocalTime` with appropriate formatter) and output time (`LocalTime` to string, again with appropriate formatter). `LocalTime` will give us much more flexibility, but indeed, it will increase complexity. Yes, saving time as a string could be quite simple way to go. String can be obtained and parsed according to specified format. However, parsing it to some instance of time (LocalDate, sql Time...) would be quite hard.
Imagine next situation: 
You saved time as a string: 22:30:20. Obvious format is: HH:mm:ss.
Now you want to parse that time in order to do some manipulation with time (like, to calculate difference between check in and check out time). You can do that quite simple since you know format (HH:mm:ss). 
However, at some point of time, you decide to save it as 22:30 or you include localization and time zones (ok that's exaggarating but still...). Now, format has changed from HH:mm:ss to HH:mm. And your whole implementation is suddenly broken. For a simple reason that your current HH:mm formatter on Java side will be unable to parse previously saved times in HH:mm:ss format.
And that's why I would go with `LocalTime` instead of SQL Time or simple string.